html/index.html

```diff
@@ -11,23 +11,27 @@
 </head>
 
 <body>
-<div id="wrapper" class="row">
-    <div id="content" class="small-12 columns">
-        <div id="target">
-        </div>
+    <div id="target">
     </div>
-</div>
 </body>
 
 <script src="../dist/vendors.bundle.js"></script>
 <script src="../dist/HomePagePanel.bundle.js"></script>
 <!-- Set to http://localhost:8080/gxa/ or http://localhost:8080/gxa_sc/ -- Remember the trailing slash! -->
 <script>
   HomePagePanel.render({
-    host: 'http://localhost:8080/gxa/sc/',
-    resource: 'json/experiments/latestExperiments',
-    panelNameAList: [`Species`, `Animals`, `Plants`, `Fungi`],
-    panelNameBList: [`Featured`, `Latest experiments`]
+    host: `http://localhost:8080/gxa/sc/`,
+    resource: {
+        Featured: `alpha/hca`,
+        Latest: `json/experiments/latestExperiments`,
+        Species: `json/experiments/popular-species`,
+        Animals: `json/experiments/popular-species?kingdom=animals`,
+        Plants: `json/experiments/popular-species?kingdom=plants`,
+        Fungi: `json/experiments/popular-species?kingdom=fungi`,
+        Protists: `json/experiments/popular-species?kingdom=protists`
+    },
+    panelNameAList: [`Species`, `Animals`, `Plants`, `Fungi`, `Protists`],
```

Comment:
A list is a data structure, it conveys no meaning about the function of this prop, or how it might be used. I think a much better alternative would be to pass pairs of a tab name with an associated resource. Notice that you want these elements to be coupled to specific URLs.